projects/wikimedia.org.yaml

```diff
@@ -42,6 +42,10 @@ paths:
             - path: ./lib/normalize_title_filter.js
               options:
                 redirect_cache_control: '{{options.purged_cache_control}}'
+            - path: lib/content_location_filter.js
+              options:
+                templates:
+                  base_uri_template: 'https://{{request.params.domain}}/api/rest_v1'
```

Comment:
In other URLs we support direct access to parameters without a prefix (just `{{domain}}`). Should we do the same here? This would require passing req.params as the global object to the template, instead of all of req.